<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="0"
            tests="0">

  <testsuite name="utilities.participantsHandler"
    errors="0" failures="0" hostname="" id="" package="utilities" skipped="0"
    tests="0" time="0.000" timestamp="">
  

  
    <system-err>
       ✖ ReferenceError
   Queue is not defined

    Error location:
    /home/nouveua/nightwatchjs/tests/dolphinVCHomePage.js:
    –––––––––––––––––––––––––––––––––––––––––––––––––––––––
     3 |     &#34;Lets connect over conference !&#34;: async function(browser) {
     4 | 
     5 |         const queue = new Queue() 
     6 |         console.log(queue.enqueue(&#39;Mahbub&#39;))
     7 |         console.log(queue.enqueue(&#39;Aditya&#39;))
    –––––––––––––––––––––––––––––––––––––––––––––––––––––––

    Stack Trace :
    at Lets connect over conference ! (/home/nouveua/nightwatchjs/tests/dolphinVCHomePage.js:5:23)
    at Context.call (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/context.js:478:35)
    at TestCase.run (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/testcase.js:58:31)
    at Runnable.__runFn (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:745:80)
    at Runnable.run (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/runnable.js:126:21)
    at TestSuite.executeRunnable (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:898:49)
    at TestSuite.handleRunnable (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:913:33)
    at /home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:745:21
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async DefaultRunner.runTestSuite (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/runner/test-runners/default.js:76:7)

 ✖ ReferenceError
   Queue is not defined

    Error location:
    /home/nouveua/nightwatchjs/tests/dolphinVCHomePage.js:
    –––––––––––––––––––––––––––––––––––––––––––––––––––––––
     3 |     &#34;Lets connect over conference !&#34;: async function(browser) {
     4 | 
     5 |         const queue = new Queue() 
     6 |         console.log(queue.enqueue(&#39;Mahbub&#39;))
     7 |         console.log(queue.enqueue(&#39;Aditya&#39;))
    –––––––––––––––––––––––––––––––––––––––––––––––––––––––

    Stack Trace :
    at Lets connect over conference ! (/home/nouveua/nightwatchjs/tests/dolphinVCHomePage.js:5:23)
    at Context.call (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/context.js:478:35)
    at TestCase.run (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/testcase.js:58:31)
    at Runnable.__runFn (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:745:80)
    at Runnable.run (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/runnable.js:126:21)
    at TestSuite.executeRunnable (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:898:49)
    at TestSuite.handleRunnable (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:913:33)
    at /home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:745:21
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async DefaultRunner.runTestSuite (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/runner/test-runners/default.js:76:7)

 ✖ ReferenceError
   Queue is not defined

    Error location:
    /home/nouveua/nightwatchjs/tests/dolphinVCHomePage.js:
    –––––––––––––––––––––––––––––––––––––––––––––––––––––––
     3 |     &#34;Lets connect over conference !&#34;: async function(browser) {
     4 | 
     5 |         const queue = new Queue() 
     6 |         console.log(queue.enqueue(&#39;Mahbub&#39;))
     7 |         console.log(queue.enqueue(&#39;Aditya&#39;))
    –––––––––––––––––––––––––––––––––––––––––––––––––––––––

    Stack Trace :
    at Lets connect over conference ! (/home/nouveua/nightwatchjs/tests/dolphinVCHomePage.js:5:23)
    at Context.call (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/context.js:478:35)
    at TestCase.run (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/testcase.js:58:31)
    at Runnable.__runFn (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:745:80)
    at Runnable.run (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/runnable.js:126:21)
    at TestSuite.executeRunnable (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:898:49)
    at TestSuite.handleRunnable (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:913:33)
    at /home/nouveua/nightwatchjs/node_modules/nightwatch/lib/testsuite/index.js:745:21
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async DefaultRunner.runTestSuite (/home/nouveua/nightwatchjs/node_modules/nightwatch/lib/runner/test-runners/default.js:76:7)

    </system-err>
  

    

  
  </testsuite>
</testsuites>
